include/sta/ConcreteNetwork.hh

```diff
@@ -206,6 +212,9 @@ public:
   Pin *connect(Instance *inst,
                LibertyPort *port,
                Net *net) override;
+  void setAttribute(Instance *inst,
+		       const char *key,
```

Comment:
indentation messed up here and many other places.
Probably an issue with tabs. I set emacs to not use tabs so the tab
setting does not matter. I tried for many many hours to configure clang-format
conform to the indentation standards and could not get it to work (even though
twenty years ago I chose the formating based on google's standard).